crates/sui-core/src/unit_tests/data/move_upgrade/additive_upgrade/sources/base.move

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module base_addr::base {
+
+    struct A<T> {
+        f1: bool,
+        f2: T
+    }
+
+    // new struct is fine
+    struct B<T> {
+        f2: bool,
+        f1: T,
+    }
+
+    friend base_addr::friend_module;
+
+    // new function is fine
+    public fun return_1(): u64 { 0 }
```

Comment:
This literally does not matter at all, but I can't help myself.

```suggestion
    public fun return_1(): u64 { 1 }
```

Reply:
You should never trust the name of functions...